1-js/02-first-steps/05-types/article.md

```diff
@@ -81,10 +81,13 @@ const bigInt = 1234567890123456789012345678901234567890n;
 
 As `BigInt` numbers are rarely needed, we don't cover them here, but devoted them a separate chapter <info:bigint>. Read it when you need such big numbers.
 
+
 ```smart header="Compatibility issues"
-Right now `BigInt` is supported in Firefox/Chrome/Edge, but not in Safari/IE.
+Right now, `BigInt` is supported in Firefox/Chrome/Edge/Safari, but not in IE.
 ```
 
+You can check [*Can I Use* BigInt compatibility table](https://caniuse.com/bigint) to know which versions of a browser are supported.
```

Comment:
Use MDN compatibility table instead of Caniuse. Caniuse is not inaccurate. but may confuse users? See https://github.com/Fyrd/caniuse/pull/5613#issuecomment-694764055 for more details.

```suggestion
You can check [*MDN* BigInt compatibility table](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Browser_compatibility) to know which versions of a browser are supported.
```